content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -104,6 +104,10 @@ Right after `kubeadm init` there should not be any pods in these states.
   likely that the Pod Network solution that you installed is somehow broken. You
   might have to grant it more RBAC privileges or use a newer version. Please file
   an issue in the Pod Network providers' issue tracker and get the issue triaged there.
+- If you install old version of docker, lower than 1.12.1, please remove the 'MountFlags=slave' option
+  when booting dockerd with systemd and restart docker. You can see it in /usr/lib/systemd/system/docker.service
+  systemd MountFlags can interfere with volumes mounted by kubernetes, then make Pods in `CrashLoopBackOff` state.
```

Comment:
`systemd` in back quotes.

kubernetes -> Kubernetes

`, then make Pods in ` -> `and put the Pods in a`.

Reply:
got it

Reply:
I can't tell where one sentence ends and the next begins.

Is it this?
You can see it in /usr/lib/systemd/system/docker.service. systemd MountFlags ...

Or is it this?
You can see it in /usr/lib/systemd/system/docker.service systemd. MountFlags ...